html/settings/_settings.scss

```diff
@@ -5,920 +5,1768 @@
+/// The line height of TypeDisplayFive.
 $sprk-line-height-display-five: 1.2 !default;
 
 // Display Six
+/// The color of the TypeDisplaySix text.
 $sprk-color-display-six: $sprk-black !default;
+/// The font-family, including fallbacks of TypeDisplaySix.
 $sprk-font-family-display-six: RocketSans, Helvetica, 'Helvetica Neue', Arial,
   sans-serif !default;
+/// The font size of TypeDisplaySix.
 $sprk-font-size-display-six: 1.25rem !default;
+/// The font weight of TypeDisplaySix.
 $sprk-font-weight-display-six: 300 !default;
+/// The line height of TypeDisplaySix.
 $sprk-line-height-display-six: 1.2 !default;
 
 // Display Seven
+/// The font-family, including fallbacks of TypeDisplaySeven.
 $sprk-font-family-display-seven: RocketSans, Helvetica, 'Helvetica Neue', Arial,
   sans-serif !default;
+/// The font size of TypeDisplaySeven.
 $sprk-font-size-display-seven: 1rem !default;
+/// The color of the TypeDisplaySeven text.
 $sprk-color-display-seven: $sprk-black !default;
+/// The line height of TypeDisplaySeven.
 $sprk-line-height-display-seven: 1 !default;
+/// The font weight of TypeDisplaySeven.
 $sprk-font-weight-display-seven: 500 !default;
 
 // Body One
+/// The font-family, including fallbacks of TypeBodyOne.
 $sprk-font-family-body-one: RocketSans, Helvetica, 'Helvetica Neue', Arial,
   sans-serif !default;
+  /// The font size of TypeBodyOne.
 $sprk-font-size-body-one: 1rem !default;
+/// The color of the TypeBodyOne text.
 $sprk-color-body-one: $sprk-black !default;
+/// The line height of TypeBodyOne.
 $sprk-line-height-body-one: 1.6 !default;
+/// The font weight of TypeBodyOne.
 $sprk-font-weight-body-one: 500 !default;
 
 // Body Two
+/// The font-family, including fallbacks of TypeBodyTwo.
 $sprk-font-family-body-two: RocketSans, Helvetica, 'Helvetica Neue', Arial,
   sans-serif !default;
+/// The font size of TypeBodyTwo.
 $sprk-font-size-body-two: 1rem !default;
+/// The color of the TypeBodyTwo text.
 $sprk-color-body-two: $sprk-black !default;
+/// The line height of TypeBodyTwo.
 $sprk-line-height-body-two: 1.6 !default;
+/// The font weight of TypeBodyTwo.
 $sprk-font-weight-body-two: 300 !default;
 
 // Body Three
+/// The font-family, including fallbacks of TypeBodyThree.
 $sprk-font-family-body-three: RocketSans, Helvetica, 'Helvetica Neue', Arial,
   sans-serif !default;
+/// The font size of TypeBodyThree.
 $sprk-font-size-body-three: 1rem !default;
+/// The color of the TypeBodyThree text.
 $sprk-color-body-three: $sprk-black !default;
+/// The line height of TypeBodyThree.
 $sprk-line-height-body-three: 2 !default;
+/// The font weight of TypeBodyThree.
 $sprk-font-weight-body-three: 300 !default;
 
 // Body Four
+/// The font-family, including fallbacks of TypeBodyFour.
 $sprk-font-family-body-four: RocketSans, Helvetica, 'Helvetica Neue', Arial,
   sans-serif !default;
+/// The font size of TypeBodyFour.
 $sprk-font-size-body-four: 0.8125rem !default;
+/// The color of the TypeBodyFour text.
```

Comment:
nit picky things: do we need to add "text" to this? the other lines don't have it.

Reply:
I removed it, thanks!